{
  "id": "MAL-2024-5255",
  "summary": "Malicious code in idtotoken (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:36:21Z",
  "published": "2024-06-25T13:36:21Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:36:21Z",
        "import_time": "2024-06-28T02:49:13.105022333Z",
        "versions": [
          "0.0.1"
        ],
        "id": "RLMA-2024-04037",
        "source": "reversing-labs",
        "sha256": "c6be8a34937ceedeedd86f672d0efbd9e40108a1608d40043fa4c3dbbefa9474"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "idtotoken",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/idtotoken"
      },
      "versions": [
        "0.0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/idtotoken/MAL-2024-5255.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}